enableEpochs.toml

```diff
@@ -312,6 +312,30 @@
     # UseGasBoundedShouldFailExecutionEnableEpoch represents the epoch when use bounded gas function should fail execution in case of error
     UseGasBoundedShouldFailExecutionEnableEpoch = 4
 
+    # DynamicESDTEnableEpoch represents the epoch when dynamic NFT feature is enabled
```

Comment:
where is this flag?
```
 # UseGasBoundedShouldFailExecutionEnableEpoch represents the epoch when use bounded gas function should fail execution in case of error
 UseGasBoundedShouldFailExecutionEnableEpoch = xxx
```

Reply:
it is on L313